projects/batfish-common-protocol/src/main/java/org/batfish/datamodel/StaticRoute.java

```diff
@@ -110,4 +110,35 @@ protected final String protocolRouteString() {
   public int routeCompare(AbstractRoute rhs) {
     return 0;
   }
+
+  public static class Builder extends AbstractRouteBuilder<StaticRoute> {
+
+    private int _administrativeCost;
+
+    private String _nextHopInterface;
+
+    @Override
+    public StaticRoute build() {
+      if (_nextHopIp == null) {
+        _nextHopIp = Route.UNSET_ROUTE_NEXT_HOP_IP;
```

Comment:
this logic should be in the class that owns `_nextHopIp`. Additionally, these variables should be `private` – children should (without a very good reason) not be able to access properties of their parents.